pyproject.toml

```diff
@@ -25,7 +25,7 @@ requires-python = ">= 3.9"
 
 # For later when model is on PyPI
 dependencies = [
-  "trachoma@git+https://github.com/NTD-Modelling-Consortium/ntd-model-trachoma",
+  "trachoma@git+https://github.com/NTD-Modelling-Consortium/ntd-model-trachoma@addOptionToTurnSurveyAndIHMEOutputOff",
```

Comment:
I'm assuming that now this is merged into main that there won't be an issue with this pointing at the branch